css/layout.css

```diff
@@ -50,24 +69,82 @@
   .column-half {
     width: 50%;
   }
+
+  .navbar {
+    width: 100vw;
+  }
```

Comment:
Don't use `vw` for setting `width`, part of the element will be hidden if a vertical scrollbar shows up. Use percentages instead: https://lfz-lms.herokuapp.com/code-guides/Learning-Fuze/curriculum/effective-css